src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
The class 'Square' should not have an empty line at the beginning of its definition. Please remove the empty line to follow best practices.